docs/src/developing/lookup-tables.md

```diff
@@ -84,9 +84,7 @@ Similar to requesting another account (or PDA) from the cluster, you can fetch a
 const lookupTableAddress = new web3.PublicKey("");
 
 // get the table from the cluster
-const lookupTableAccount = await connection
-  .getAddressLookupTable(lookupTableAddress)
-  .then((res) => res.value);
+const lookupTableAccount (await connection.getAddressLookupTable(lookupTableAddress)).value;
```

Comment:
I guess the problem with the old code is that `lookupTableAccount` might be `null`. I wonder if it it would be better to do explicit the check for `null` or for the documentation purposes it make text to be harder to comprehend

Reply:
Actually, another issue was causing me to get `null`.

But I still think it's easier to comprehend not mixing `await` and `.then`. But I can change this part back if you still prefer the existing code.

Reply:
Either method is a valid method of handling the value returned from a Promise in JS/TS. It's really just a matter of style and preference. The `.then()` methodology allows for better, more explicit error handling of the specific Promise (using `.catch()`) without having to wrap everything in one or many try/catch blocks. 

As written, your suggested change @loganyu would result in the same error if no lookup table was found. In fact, you add a different error since you forgot the `=` when creating your variable :) 

But also as a general statement, this code example in this doc is meant to give a quick reference on how someone could use the web3js library to interact with lookup tables. So it is not really meant to handle every possible error or situation. It is merely a quick reference. Having loads of extra "checks" in the doc references adds a lot to them and makes them generally harder for people to navigate

Reply:
Thanks for the details Nick! Good catch on the missing `=` 😅 I fixed it.